phoenicis-library/src/main/java/org/phoenicis/library/ShortcutManager.java

```diff
@@ -105,11 +109,16 @@ public void deleteShortcut(ShortcutDTO shortcutDTO) {
         final String baseName = shortcutDTO.getName();
         final File shortcutDirectory = new File(this.shortcutDirectory);
 
+        final File categoryFile = new File(shortcutDirectory, baseName + ".category");
```

Comment:
I think we need a definition in the wiki about what these files (`.category`, `.shortcut`, `.icon`, `.miniature`, `.description`) contain.